src/olympia/editors/views.py

```diff
@@ -534,18 +534,56 @@ def application_versions_json(request):
     return {'choices': f.version_choices_for_app_id(app_id)}
 
 
+def _get_deleted_versions(addon):
```

Comment:
opted for text rather than a dragon: https://github.com/mozilla/addons-server/pull/4048/commits/fdf22a8c00a4af888322daf51488bdba2d9a83d6